### ISSUE TITLE
mmdet3d coordinate for gt boxes are `xyzlwh`, here in CenterFormer, gtboxes[3] should be length not width.

### ISSUE THREAD

**Issue by DezeZhao** [open-mmlab/mmdetection3d#2360]
### Prerequisite

- [X] I have searched [Issues](https://github.com/open-mmlab/mmdetection3d/issues) and [Discussions](https://github.com/open-mmlab/mmdetection3d/discussions) but cannot get the expected help.
- [X] I have read the [FAQ documentation](https://github.com/open-mmlab/mmdetection3d/blob/1.1/docs/en/notes/faq.md) but cannot get the expected help.
- [X] The bug has not been fixed in the [latest version (dev)](https://github.com/open-mmlab/mmdetection3d/tree/dev) or [latest version (1.x)](https://github.com/open-mmlab/mmdetection3d/tree/dev-1.x).

### Task

I'm using the official example scripts/configs for the officially supported tasks/models/datasets.

### Branch

master branch https://github.com/open-mmlab/mmdetection3d

### Environment

no

### Reproduces the problem - code sample

https://github.com/open-mmlab/mmdetection3d/blob/341ff9980f624b2fbd3ce2b161590f821056f0a3/projects/CenterFormer/centerformer/centerformer_backbone.py#L662

### Reproduces the problem - command or script

https://github.com/open-mmlab/mmdetection3d/blob/341ff9980f624b2fbd3ce2b161590f821056f0a3/projects/CenterFormer/centerformer/centerformer_backbone.py#L662

### Reproduces the problem - error message

https://github.com/open-mmlab/mmdetection3d/blob/341ff9980f624b2fbd3ce2b161590f821056f0a3/projects/CenterFormer/centerformer/centerformer_backbone.py#L662

### Additional information

https://github.com/open-mmlab/mmdetection3d/blob/341ff9980f624b2fbd3ce2b161590f821056f0a3/projects/CenterFormer/centerformer/centerformer_backbone.py#L662

**Comment by JingweiZhang12** [comment]
@DezeZhao， Hi, thanks for your kind reminder.  It's a minor flaw.  Fortunately, it has not related to the precision considering the detailed implements among this: https://github.com/open-mmlab/mmdetection3d/blob/341ff9980f624b2fbd3ce2b161590f821056f0a3/projects/CenterFormer/centerformer/centerformer_backbone.py#L670.  
Anyway, could you create a PR to fix it (include here: https://github.com/open-mmlab/mmdetection3d/blob/341ff9980f/mmdet3d/models/dense_heads/centerpoint_head.py#L522)

**Comment by DezeZhao** [comment]
yeah, I got it later. It is ok if these 2 values corresponding to x and y axis and keep their order.

**Comment by selmadeac** [comment]
Are you sure this is an issue? length should be larger than width, right?
If i print this in the original implementation it is ok. 

 if legth>width:
         print("TRUE")

**Comment by DezeZhao** [comment]
In mmdetection3d, **gt boxes organized by `xyzlwhr` order**, and length may not always more than width.
